common/app/views/support/GetClasses.scala

```diff
@@ -41,7 +41,7 @@ object GetClasses {
       ("fc-item--is-commentable", item.discussionSettings.isCommentable),
       ("fc-item--is-media-link", item.isMediaLink),
       ("fc-item--has-video-main-media", item.hasVideoMainMedia),
-      ("fc-item--dynamic-layout", item.cardTypes.canBeDynamicLayout && !item.cutOut.isDefined)
+      ("fc-item--dynamic-layout", isDynamic && item.cardTypes.canBeDynamicLayout && !item.cutOut.isDefined)
```

Comment:
I'm not sure if we could now lose some of the additional conditions here, any thoughts?

Reply:
Think it looks like it'd be ok, we do only want it to apply to that specific dynamoContentCard!